system/modules/core/elements/ContentAlias.php

```diff
@@ -39,7 +39,7 @@ public function generate()
 			return '';
 		}
 
-		// Overwrite the ID
+		/** @var \ContentElement $objElement */
```

Comment:
Why is this necessary? `\ContentModel::findByPk` should already know the correct type?